spacelift/resource_run.go

```diff
@@ -55,16 +67,78 @@ func resourceRun() *schema.Resource {
 				Type:        schema.TypeString,
 				Computed:    true,
 			},
+			"wait": {
+				Type:        schema.TypeList,
+				Optional:    true,
+				Description: "Wait for the run to finish",
+				MaxItems:    1,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"disabled": {
+							Type:        schema.TypeBool,
+							Description: "Whether waiting for a job is disabled or not. Default: `false`",
+							Optional:    true,
+							Default:     false,
+						},
+						"continue_on_state": {
+							Type: schema.TypeSet,
+							Elem: &schema.Schema{
+								Type: schema.TypeString,
+							},
+							Description: "Continue on the specified states of a finished run. If not specified, the default is `[ 'finished' ]`",
```

Comment:
can we give a few more tips how many states do we have? because I'm not sure tbh :D
```suggestion
							Description: "Continue on the specified states of a finished run. If not specified, the default is `[ 'finished' ]`. You can use the following state: x, y, z.",
```